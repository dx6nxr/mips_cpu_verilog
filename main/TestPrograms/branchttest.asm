jal func
lui $2, 0x1234
ori $2, 100
lui $3, 0x0000
ori $3, 238
multu $2, $3
mfhi $2
mflo $4
j exit

func:
addiu $5, $0, 7
lui $6, 0x0000
ori $6, 5
jr $ra

exit: