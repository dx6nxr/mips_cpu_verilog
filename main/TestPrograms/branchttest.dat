0c100009
3c021234
34420064
3c030000
346300ee
0043001b
00001010
00002012
0810000d
24050007
3c060000
34c60005
03e00008