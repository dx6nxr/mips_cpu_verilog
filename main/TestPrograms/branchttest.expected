@2 00000044
@3 000000ee
@4 00139470
@5 00000007
@6 00000005
@1f 00400004